lib/modules/manager/mix/extract.ts

```diff
@@ -6,7 +6,7 @@ import type { PackageDependency, PackageFile } from '../types';
 
 const depSectionRegExp = regEx(/defp\s+deps.*do/g);
 const depMatchRegExp = regEx(
-  /{:(?<depName>\w+),\s*(?<datasource>[^:"]+)?:?\s*"(?<currentValue>[^"]+)",?\s*(?:organization: "(?<organization>.*)")?.*}/gm
+  /[^#]{:(?<depName>\w+),\s*(?<datasource>[^:"]+)?:?\s*"(?<currentValue>[^"]+)",?\s*(?:organization: "(?<organization>.*)")?.*}/gm
```

Comment:
🤔 is `#` a comment line character? so we can simply skip all lines if they start with `/^\s*#/m` ?

Reply:
I think so. Do you have in mind some cases where this might cause problems?

Reply:
yes, that regex requires any non `#` char before depName. so when there is no space before it doesn't match. not sure if this can happen.

Reply:
True, it works but isn't accurate.

Reply:
One thing still remains:
To handle commets like these:

```exs
 {:postgrex, "~> 0.8.1"},    #  {:broadway_dashboard, "~> 0.2.2"},
```